uaclient/tests/test_util.py

```diff
@@ -727,3 +727,74 @@ class TestRedactSensitiveLogs:
     def test_redact_all_matching_regexs(self, raw_log, expected):
         """Redact all sensitive matches from log messages."""
         assert expected == util.redact_sensitive_logs(raw_log)
+
+
+class TestWrapText:
```

Comment:
This only looks like a huge test because it involves long strings :upside_down_face: